styles.css

```diff
@@ -90,6 +104,15 @@ button {
   }
 }
 
+@media screen and (max-width: 953px) {
```

Comment:
This doesn't actually seem to do anything. I think it has to do with the viewport being set to 980px by default so media queries below that are ignored. I actually tried to change the viewport to match the screensize - but I found some issues with that (maybe partially due to my high DPI screen).

I think for now it's fine with flex direction row, I might make a section above the header in the future with a select box for the gen, and move this button there.

Reply:
that's odd that it doesn't work for you, cause the reason I put that there was because it was working for me!  Lol.  But, also I see you don't have <meta name="viewport" content="width=device-width, initial-scale=1.0"> included in the <head> and not having that sometimes makes media queries act a little weird.  but, definitely don't add that in now because then it'll make the table all messed up.

Reply:
but also, instead of doing flex-direction column in a media query, we could try just putting flex-wrap on it so it wraps automatically when it needs to.  then it won't be dependent on the viewport size

Reply:
update: tried flex-wrap and you're right.  it's for sure fine with flex-direction row for now :D